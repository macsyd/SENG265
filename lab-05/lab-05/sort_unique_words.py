#! /usr/bin/env python3

import sys

def main():
    
    #  Read the words from sys.stdin
    #  Keep only the unique words
    #  Print them in sorted order.
    lst = []
    for line in sys.stdin:
        new_line = line.split('\n')
        words = new_line[0].split(',')
        for each in words:
            lst.append(each)
    
    lst = set(lst)
    lst = list(lst)
    lst.sort()
    for word in lst:
        print(word)

if __name__ == "__main__":
    main()
