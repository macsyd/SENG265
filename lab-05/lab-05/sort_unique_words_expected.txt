acute
apparent
balance
barbigerous
blatant
blue
bounce
brainy
bright
brilliant
bump
cheerful
clean
clear
clever
crinite
crinose
crude
cutting
definite
delighted
dim
distinct
edged
equable
equalize
even
evident
faint
feeble
fine
flaring
garish
gentl
glad
glaring
hairy
happy
harmonize
hirsute
hustle
jar
jostle
jounce
keen
knifelike
loud
low
manifest
obstrusive
obvious
ostensible
pain
palpable
patent
perpicuous
plain
pleased
plumb
poignant
pure
regular
screaming
self-evident
sharp
sheer
smart
soft
symmetrize
uniform
veritable
vulgar
weak
